megamek/src/megamek/common/LandAirMech.java

```diff
@@ -127,6 +127,13 @@ public static int getAeroLocation(int loc) {
     private int straightMoves = 0;
     private int altLoss = 0;
     private int altLossThisRound = 0;
+    
+    //Autoejection
+    private boolean autoEject = true;
+    private boolean condEjectAmmo = true;
+    private boolean condEjectFuel = true;
```

Comment:
Applies only to Aero critical hits. This is never used for LAMs.

Reply:
Fixed